#msg_MapPoint
int64 id_MapPoint
geometry_msgs/Point position 	#(x,y,z)
geometry_msgs/Point normal
float32[] descriptor
float64[] covariance